### DISCUSSION TITLE
Changing footers for specific wiki pages

### DISCUSSION THREAD

**Discussion by mathomp4** [community/community#103040] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I was wondering if there is a way to change or suppress the default footer from `_Footer.md` on GitHub repo wikis? 

For example, on a repo I manage, most of the wiki pages are edited and maintained by one group, say Team A, and so the footer says "need help, contact Team A". But, there are other pages that are maintained by Team B, and we'd like the footer on those pages to say "need help, contact Team B".

However, we can't figure out how to do this. You change `_Footer.md` and every page changes.

**Comment by gil-son** [comment]
You have to modify file by file or create a programming to do that:

```
import os
import re

wiki_path = 'path/to/your/wiki'
team_a_footer = 'For assistance, please contact Team A.'
team_b_footer = 'For assistance, please contact Team B.'

for root, dirs, files in os.walk(wiki_path):
    for file in files:
        if file.endswith('.md'):
            file_path = os.path.join(root, file)

            # Read the file content
            with open(file_path, 'r', encoding='utf-8') as f:
                content = f.read()

            # Modify the footer based on conditions
            if 'Team A' in content:
                content = re.sub(r'## Need Help.*', f'## Need Help\n\n{team_a_footer}', content)
            elif 'Team B' in content:
                content = re.sub(r'## Need Help.*', f'## Need Help\n\n{team_b_footer}', content)

            # Write the modified content back to the file
            with open(file_path, 'w', encoding='utf-8') as f:
                f.write(content)

```

In this case I illustrated using python, but you can use your preference programming language
